src/libvirtApi/common.js

```diff
@@ -411,15 +419,15 @@ function startEventMonitorStoragePools(connectionName) {
         });
 }
 
-function storagePoolUpdateOrDelete(connectionName, poolPath) {
+function storagePoolStopOrUndefine(connectionName, poolPath) {
     return call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "ListStoragePools", [0], { timeout, type: "u" })
             .then(objPaths => {
                 if (objPaths[0].includes(poolPath))
                     return storagePoolGet({ connectionName, id: poolPath, updateOnly: true });
                 else // Transient pool which got undefined when stopped
                     return store.dispatch(undefineStoragePool({ connectionName, id: poolPath }));
             })
-            .catch(ex => console.warn("storagePoolUpdateOrDelete action failed:", ex.toString()));
+            .catch(ex => console.warn("storagePoolStopOrUndefine action failed:", ex.toString()));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1285-20231102-144208-464cf7a9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)